shell/browser/common_web_contents_delegate.cc

```diff
@@ -292,8 +292,21 @@ content::WebContents* CommonWebContentsDelegate::OpenURLFromTab(
   load_url_params.should_replace_current_entry =
       params.should_replace_current_entry;
   load_url_params.is_renderer_initiated = params.is_renderer_initiated;
+  load_url_params.started_from_context_menu = params.started_from_context_menu;
   load_url_params.initiator_origin = params.initiator_origin;
-  load_url_params.should_clear_history_list = true;
```

Comment:
We maintain our own navigation controller, so its necessary to clear the chromium history list.

Reply:
The tests aren't failing. How would I detect that this is causing incorrect behavior?

Reply:
FWIW removing this fixed a `DCHECK(!replace)` in the navigation controller.

Reply:
Hmm the non-sandboxed renderer history are unaffected by this change, tested with:

* load `file://index.html`
* window.history.length == 1
* window.location.reload() // creates new process
* window.location = "https://example.com" // creates new process
* window.location.reload() // creates new process
* window.history.length == 4 but should be 2 , this bug is not related to this change.

So seems like custom navigation controller still takes control for non-sandboxed renderers.

Reply:
Sounds like we're all OK on this front :)